deployment/live/gcp/static-ct-staging/logs/arche2025h1/terragrunt.hcl

```diff
@@ -0,0 +1,21 @@
+terraform {
+  source = "${get_repo_root()}/deployment/modules/gcp//tesseract/conformance"
+}
+
+locals {
+  env                 = include.root.locals.env
+  docker_env          = local.env
+  base_name           = include.root.locals.base_name
+  origin_suffix       = include.root.locals.origin_suffix
+  server_docker_image = "us-central1-docker.pkg.dev/${include.root.locals.project_id}/docker-${local.env}/conformance-gcp:${include.root.locals.docker_container_tag}"
```

Comment:
`us-central1` should be replaced with the location environment variable.

Reply:
Added this to static-ct as well